salt/core/fail2ban/init.sls

```diff
@@ -0,0 +1,24 @@
+include: 
+  - core.mail
+
+fail2ban:
+  pkg.installed
+
+f2b-startup:
+  service.running:
+    - name: fail2ban
+    - enable: True
+    - reload: True
+  require:
+    - pkg: fail2ban
+    - sls: core.mail
+  watch:
+    - file: /etc/fail2ban/jail.local
+
+/etc/fail2ban/jail.local:
+  file.managed:
+    - source: salt://core/fail2ban/jail.local
+
+# In the future I'd like to whitelist both admin and server IP's as well.
+# grains.get ip_interfaces
```

Comment:
Let's create an issue and delete the comment. Comments aren't visible for task prioritization.

Reply:
I've removed this comment. I will come to you with a number of future enhancements I've collected together at a later point. Then we can decide if we want to make this into an issue.